pages/styles/challenge.module.css

```diff
@@ -22,3 +22,15 @@
 .Challenge .centerText {
   text-align: center;
 }
+
+.Challenge .challengerListColumn {
+  padding: 0;
+  align-self: flex-start;
+  flex-basis: 225px;
+  list-style-position: inside;
+}
+
+.Challenge .challengerListContainer div {
+  width: 99vw;
```

Comment:
Hmmm this wont work cuz - as you can see - on mobile views there's a horizontal scrollbar. What you'll actually wann do is pass a className to `Content` which does `width: 100%` and then apply that same property to the empty div on line 199 + `.challengerListContainer`

Reply:
![Screen Shot 2020-07-31 at 1 57 59 PM](https://user-images.githubusercontent.com/9523719/89076833-dc3e0180-d335-11ea-8e9f-e2403e60b046.png)
like so

Reply:
As we discussed in Slack, the Content component className is not exposed so for now the fix will be to just lower width to 85vw. In the future we can expose the className for the Content component in order to change width to 100% as mentioned.